lib/src/hidden_ranges/line_numbering_breakpoint.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:equatable/equatable.dart';
+
+class LineNumberingBreakpoint with EquatableMixin {
+  /// The full line index.
+  final int full;
```

Comment:
Suggest to name it as `fullLineIndex` and `visibleLineIndex`, so comments will be redundant

Reply:
I believe `LineNumbering...` in the class name gives enough context to `full`.
At the same time `visibleLineIndex` would still call for a comment.
Also longer names would cause excessive line breaks.